test/index.js

```diff
@@ -183,3 +183,171 @@ describe('metalsmith-serve with custom indexFile', function(){
 
 });
 
+
+// not_found file serving and redirects
+describe('metalsmith-serve', function() {
```

Comment:
Could you please rename this suite or merge it with the first suite that shares the same name?